packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
```

Comment:
_Suggestion_ Since you're just merging the key and the src into a single URL, I would just use a single parameter.